store.go

```diff
@@ -26,12 +26,20 @@ var (
 	ErrTraceNotFound = errors.New("trace not found")
 )
 
+type TracesOpts struct {
+	Timespan Timespan
+}
+
+func DefaultTracesOpts() TracesOpts {
+	return TracesOpts{}
+}
+
 // A Queryer indexes spans and makes them queryable.
 type Queryer interface {
 	// Traces returns an implementation-defined list of traces. It is
 	// a placeholder method that will be removed when other, more
 	// useful methods are added to Queryer.
-	Traces() ([]*Trace, error)
+	Traces(opts TracesOpts) ([]*Trace, error)
```

Comment:
Def. agree on updating the `Queryer.Traces(...)`, addressed on https://github.com/sourcegraph/appdash/pull/127/files#diff-1ac94134b8964626aaac00790fa9b48fR37.

***

I think `opts *TraceOpts` seems appealing alternative but IMO we should go for `opts TraceOpts` since the first one introduces a couple of potential issues:
  - Ambiguity on if `Traces(nil)` & `Traces(&TracesOpts{})` have the same/diff behavior.
  - Lib users could wonder about what happens if `&TracesOpts{...}` is mutated after calling `Traces(&TracesOpts{...})`.
 
 We could have even more sophisticated API like `functional options` but I might be an overkill, also the points mentioned above are pretty good explained by Dave Cheney [blog post](http://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis).